tests/cases/phiself.py

```diff
@@ -3,4 +3,4 @@
 # University of Illinois/NCSA Open Source License.  Both these licenses can be
 # found in the LICENSE file.
 
-assert '$phi' not in generated, 'we should optimize out self-phis'
+assert '$phi' not in generated, 'we should optimize out self-phis'  # noqa
```

Comment:
just curious, what didn't it like about this line?

Reply:
generated not defined.. its because we use "eval()" in the test code.  Its not nice, and this is the only .py file in this directory, so we could probably remove this..  but for now I'll leave it alone.